DemoApp/App.tsx

```diff
@@ -83,19 +83,29 @@ function App(): JSX.Element {
               },
             )
           }}>
-          <Text style={styles.text}> Start SDK </Text>
+          <Text testID={'init'} style={styles.text}>
+            Start SDK
+          </Text>
```

Comment:
Adding `testId` was mandatory as otherwise XCUITest would not find the `Start SDK` text.